### PR TITLE
refactor/ Paper Trade Rework

### PR DESCRIPTION
**Pull request by CrimsonJacket**
**Before submitting this PR, please make sure**:

* [x] Your code builds clean without any errors or warnings
* [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

* [x] paper_trade command removed
* [x] paper_trade_mode ticker removed
~~* [ ] New paper_trade_exchanges global config added~~

**Tests performed by the developer**:
Numerous changes throughout Hummingbot Client to include Paper Trade support for selected exchanges.
Changes are mainly on the `commands`, `global_config_map` and the `hummingbot_application.py` files
Added `AllConnectorSettings` class object.
Refactor various components to dynamically call `AllConnectorSettings` to retrieve connector settings.

**Tips for QA testing**:
Any strategy that works with spot exchange (connector) works with the following exchanges

* `binance_paper_trade`
* `kucoin_paper_trade`
* `ascend_ex_paper_trade`
* `gate_io_paper_trade`

┆Issue is synchronized with this [Clickup task](https://app.clickup.com/t/1rvw2kh) by [Unito](https://www.unito.io)

### CODE REVIEW

File: hummingbot/client/settings.py [hunk 1]
```diff
@@ -201,13 +262,7 @@ def ethereum_required_trading_pairs() -> List[str]:
...
 
-CONNECTOR_SETTINGS = _create_connector_settings()
-DERIVATIVES = {cs.name for cs in CONNECTOR_SETTINGS.values() if cs.type is ConnectorType.Derivative}
-EXCHANGES = {cs.name for cs in CONNECTOR_SETTINGS.values() if cs.type is ConnectorType.Exchange}
-OTHER_CONNECTORS = {cs.name for cs in CONNECTOR_SETTINGS.values() if cs.type is ConnectorType.Connector}
-ETH_WALLET_CONNECTORS = {cs.name for cs in CONNECTOR_SETTINGS.values() if cs.use_ethereum_wallet}
-ALL_CONNECTORS = {"exchange": EXCHANGES, "connector": OTHER_CONNECTORS, "derivative": DERIVATIVES}
-EXAMPLE_PAIRS = {name: cs.example_pair for name, cs in CONNECTOR_SETTINGS.items()}
-EXAMPLE_ASSETS = {name: cs.example_pair.split("-")[0] for name, cs in CONNECTOR_SETTINGS.items()}
+EXAMPLE_PAIRS = {name: cs.example_pair for name, cs in AllConnectorSettings.get_connector_settings().items()}
```
Comment (Nullably) [inline hummingbot/client/settings.py:265]: We don't need to keep these 2 constants (they can be class method like all the others). Let's keep it clean, keep constants for only primitive values.

**Review by Nullably — CHANGES_REQUESTED** [review]

**Comment by dennisocana** [comment]
@RC-13 QA can start testing the PR

**Comment by bluecammut** [comment]
LGTM

**Comment by RC-13** [comment]
When using any of the paper trade connector and aroon-oscillator strategy, the following error comes up and preventing the strategy to start:
```python
2021-11-02 04:42:20,387 - 15797 - hummingbot.client.hummingbot_application - ERROR - Unknown error during initialization.
Traceback (most recent call last):
  File "/home/ubuntu/paper/hummingbot/strategy/aroon_oscillator/start.py", line 55, in start
    self._initialize_wallet(token_trading_pairs=list(set(maker_assets)))
AttributeError: 'HummingbotApplication' object has no attribute '_initialize_wallet'
2021-11-02 04:42:20,388 - 15797 - hummingbot.client.hummingbot_application - INFO - start command initiated.
```
image

**Review by Nullably — APPROVED** [review]
LGTM

**Review by rapcmia — APPROVED** [review]
Tested and confirmed ok. Approved on behalf of QA

**Pull request merged**